exec/java-exec/src/test/java/org/apache/drill/exec/fn/impl/TestNewSimpleRepeatedFunctions.java

```diff
@@ -17,36 +17,161 @@
  */
 package org.apache.drill.exec.fn.impl;
 
-import org.apache.drill.test.BaseTestQuery;
 import org.apache.drill.categories.SqlFunctionTest;
+import org.apache.drill.test.ClusterFixture;
+import org.apache.drill.test.ClusterTest;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+
 @Category(SqlFunctionTest.class)
-public class TestNewSimpleRepeatedFunctions extends BaseTestQuery {
+public class TestNewSimpleRepeatedFunctions extends ClusterTest {
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    startCluster(ClusterFixture.builder(dirTestWatcher));
+  }
+
   @Test
   public void testRepeatedContainsForWildCards() throws Exception {
-     testBuilder().
-       sqlQuery("select repeated_contains(topping, 'Choc*') from cp.`testRepeatedWrite.json`")
-       .ordered()
-       .baselineColumns("EXPR$0")
-       .baselineValues(true)
-       .baselineValues(true)
-       .baselineValues(true)
-       .baselineValues(true)
-       .baselineValues(false)
-       .build().run();
-
-     testBuilder().
-       sqlQuery("select repeated_contains(topping, 'Pow*') from cp.`testRepeatedWrite.json`")
-       .ordered()
-       .baselineColumns("EXPR$0")
-       .baselineValues(true)
-       .baselineValues(false)
-       .baselineValues(false)
-       .baselineValues(true)
-       .baselineValues(false)
-       .build().run();
+    testBuilder()
+        .sqlQuery("select repeated_contains(topping, 'Choc*') from cp.`testRepeatedWrite.json`")
+        .ordered()
+        .baselineColumns("EXPR$0")
+        .baselineValuesForSingleColumn(true, true, true, true, false)
+        .go();
+
+    testBuilder()
```

Comment:
It is better to divide these big tests into smaller one. Usually one `testBuilder` per test case.